src/Forms/ValidationList/service.js

```diff
@@ -0,0 +1,35 @@
+import {
+  defaultTo,
+  flow,
+  identity,
+  keys,
+  reduce,
+  pickBy,
+  includes
+} from 'lodash/fp';
+
+export const getActiveErrors = flow(pickBy(identity), keys);
+
+export const getErrorMessages = ({ data, messages = {}, translateFn } = {}) =>
+  flow(
+    defaultTo([]),
+    reduce((activeMsgs, error) => {
+      const message = messages[error];
+
+      if (!message) {
+        return activeMsgs;
+      }
+
+      const messageType = typeof message;
+
+      if (!includes(messageType, ['string', 'function'])) {
+        throw new TypeError(
```

Comment:
I know I wrote this. Question, how do we feel about throwing/error handling? I recently read another tweet arguing against throwing. Can't recall why though. But in particular in a library like this, we should probably have an opinion an be consistent about it. 😁